notifier.rst

```diff
@@ -159,6 +159,9 @@ Service
 `Sinch`_            **Install**: ``composer require symfony/sinch-notifier`` \
                     **DSN**: ``sinch://ACCOUNT_ID:AUTH_TOKEN@default?from=FROM`` \
                     **Webhook support**: No
+`Sipgate`_          **Install**: ``composer require symfony/sipgate-notifier`` \
```

Comment:
Please add a link at the end of the page, like done for the other bridges

+ add your bridge to the versionadded directive below this table, thanks

Reply:
Sorry I dont see what you mean. A link to what? To the symfony repo where the bridge will be located? :-)

Reply:
If you open this file and scroll to the bottom you will see it I guess 😃 

`Sipgate`_ means a link, and the link is defined down below

Reply:
Ah sure thing, I got you, thanks! Now it's added.